samples/logback.xml

```diff
@@ -11,6 +11,15 @@
         <apiKey>YOUR_API_KEY_HERE</apiKey>
         <logfile>myapp</logfile>
         <parser>logback</parser>
+        <serverHost>myhost</serverHost>
+        <!-- Uncomment to set an env and send to scalyr in the ServerInfo
```

Comment:
How about:

Uncomment to attach an "env" field to each log message sent to Scalyr. This can be used for grouping or filtering messages by the environment that generated them.